src/components/Menu.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import ".././styles/styles.scss";
+
+class Menu extends React.Component<any, any> {
+    constructor(props) {
+        super(props);
+        this.state = {activeMenuItem : "(none)"};
```

Comment:
this is used here to put some sense ![below](https://github.com/aefox/intra-hq/pull/7/files#diff-1171f0d991a0986059d76356c1d22a2fR35) if there's no menu item that's active. but `"(none)"` actually does not contribute to anything else in this component. I would have gone here with either an empty object or a `null` value for the `activeMenuItem` property